packages/webapp/src/routes/land/add-legal-agreement-parties.js

```diff
@@ -158,19 +162,25 @@ const handlers = {
       inProgressUrl: constants.routes.ADD_LEGAL_AGREEMENT_PARTIES
     })
     const partySelectionData = request.yar.get(constants.redisKeys.LEGAL_AGREEMENT_PARTIES)
+    const documentType = request.yar.get(constants.redisKeys.LEGAL_AGREEMENT_DOCUMENT_TYPE)?.toLowerCase()
```

Comment:
Would it be better to get the legalAgreementType here? So as not to duplicate the getLegalAgreementDocumentType(documentType) in the return h.view...

Only a could

Reply:
agree. I have applied same change for all files.